ouroboros-consensus/test/consensus-test/Test/Consensus/Ledger/Tables/Diff.hs

```diff
@@ -54,24 +71,89 @@ lawsTestOne p tts =
   Applying diffs
 ------------------------------------------------------------------------------}
 
-type K = Int
-type V = Char
+type K = Word16
+type V = Word16
```

Comment:
The arbitrary instance or `Word16` generates a value uniformly over the whole range `[minBound .. maxBound]`, so the probability of generating the same key twice is very small. I think you should use a generator with a higher probability of duplicates to see if the tests still pass